### DISCUSSION TITLE
Sortable not working in app production hosted on Vercel

### DISCUSSION THREAD

**Discussion by flexa-david** [clauderic/dnd-kit#782] [Q&A]
For some reason, the onDragStart does not fire on the deployed version, however is working fine in dev, and the built version (ts next.js), however once deployed, the sortable no longer works and clicking and dragging on the handle has no effect. 

Any ideas?

Update: 

It appears as though the isDragging condition is rendered serverside as true. The page is rendered as if the person is clicking and holding on the drag handle. Not sure if this is something to do with the useSortable hook and SSR

**Comment by clauderic** [comment]
Hey @flexa-david, could you provide a replication case? Ideally on CodeSandbox or Github CodeSpaces so I don't need to set up anything locally on my machine.

**Comment by flexa-david** [comment]
Hi @clauderic, thanks for your response - turns out it was a data issue on my side. For what it's worth, it was because the `id` that was being used in the `useSortable` was undefined (perhaps in SSR), and it was stuck in isDragging === true, due to undefined === undefined in https://github.com/clauderic/dnd-kit/blob/8fbbfd79df7ff9343017f3067d04f1636a745f02/packages/core/src/hooks/useDraggable.ts#L61 .
